### PR TITLE
[CAFV-216][Docs]TKG version updates and minor edits in the docs

### PR DESCRIPTION
**Pull request by sahithi**
## Description
Please provide a brief description of the changes proposed in this Pull Request

- 

## Checklist
- [ ] tested locally
- [ ] updated any relevant dependencies
- [ ] updated any relevant documentation or examples

## API Changes
Are there API changes?
- [ ] Yes
- [ ] No

If yes, please fill in the below

1. Updated conversions?
    - [ ] Yes
    - [ ] No
    - [ ] N/A
2. Updated CRDs?
    - [ ] Yes
    - [ ] No
    - [ ] N/A
3. Updated infrastructure-components.yaml?
    - [ ] Yes
    - [ ] No
    - [ ] N/A
4. Updated `./examples/capi-quickstart.yaml`?
    - [ ] Yes
    - [ ] No
    - [ ] N/A
5. Updated necessary files under `./infrastructure-vcd/v1.0.0/`?
   - [ ] Yes
   - [ ] No
   - [ ] N/A

## Issue
If applicable, please reference the relevant issue

Fixes #

- - -
This change is [Reviewable](https://reviewable.io/reviews/vmware/cluster-api-provider-cloud-director/473)

### CODE REVIEW

**Review by lzichong — APPROVED** [review]
> Reviewed 2 of 2 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/vmware/cluster-api-provider-cloud-director/473)* status: all files reviewed, 2 unresolved discussions (waiting on @Anirudh9794 and @sahithi)

___
*[`README.md` line 17 at r1](https://reviewable.io/reviews/vmware/cluster-api-provider-cloud-director/473#-NZhCBolBVyRtq-Sv_sA:-NZhC2OwBlB4_p_QH0zo:bgdargn) ([raw file](https://github.com/vmware/cluster-api-provider-cloud-director/blob/7113ba4575140ed8ed7ddaa47d0f08697f4bf540/README.md#L17)):*
> ```Markdown
> |                                  CAPVCD Version                                   | VMware Cloud Director API |                              VMware Cloud Director Installation                              |                                                                CoreCAPI/Clusterctl CLI version                                                                 | Kubernetes Versions                                                                         | CSI Versions                                                                                                                                                                                                                                                                                                        | CPI Versions                                                                                                                                                                                                                                                                                                  | 
> |:---------------------------------------------------------------------------------:|:-------------------------:|:--------------------------------------------------------------------------------------------:|:--------------------------------------------------------------------------------------------------------------------------------------------------------------:|:--------------------------------------------------------------------------------------------|:--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
> |  [main](https://github.com/vmware/cluster-api-provider-cloud-director/tree/main)  |           36.0+           | 10.3.1+ <br/>(10.3.1 needs hot-patch to prevent VCD cell crashes in multi-cell environments) |                                          [1.4.0](https://github.com/kubernetes-sigs/cluster-api/releases/tag/v1.4.0)                                           | <ul><li>1.25</li><ul><li>1.24</li><li>1.23</li><li>1.22</li><li>1.21</li><li>1.20</li></ul> | <ul><li>[1.4](https://github.com/vmware/cloud-director-named-disk-csi-driver/releases/tag/1.4)</li> <ul><li>[1.3.1](https://github.com/vmware/cloud-director-named-disk-csi-driver/releases/tag/1.3.1)</li><li>[1.3.0](https://github.com/vmware/cloud-director-named-disk-csi-driver/releases/tag/1.3.0)</li></ul> | <ul><li>[1.4.0](https://github.com/vmware/cloud-provider-for-cloud-director/releases/tag/1.4.0)</li><ul><li>[1.3.0](https://github.com/vmware/cloud-provider-for-cloud-director/releases/tag/1.3.0)</li><li>[1.2.0](https://github.com/vmware/cloud-provider-for-cloud-director/releases/tag/1.2.0)</li></ul> |
> ```

nit: There is a nested `<ul>` tag, causing the 1.24 versions and below to be indented after v1.25
___
*[`README.md` line 17 at r1](https://reviewable.io/reviews/vmware/cluster-api-provider-cloud-director/473#-NZhCZUG8DTTAEiTNUzT:-NZhCZUG8DTTAEiTNUzU:b2odt27) ([raw file](https://github.com/vmware/cluster-api-provider-cloud-director/blob/7113ba4575140ed8ed7ddaa47d0f08697f4bf540/README.md#L17)):*
> ```Markdown
> |                                  CAPVCD Version                                   | VMware Cloud Director API |                              VMware Cloud Director Installation                              |                                                                CoreCAPI/Clusterctl CLI version                                                                 | Kubernetes Versions                                                                         | CSI Versions                                                                                                                                                                                                                                                                                                        | CPI Versions                                                                                                                                                                                                                                                                                                  | 
> |:---------------------------------------------------------------------------------:|:-------------------------:|:--------------------------------------------------------------------------------------------:|:--------------------------------------------------------------------------------------------------------------------------------------------------------------:|:--------------------------------------------------------------------------------------------|:--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
> |  [main](https://github.com/vmware/cluster-api-provider-cloud-director/tree/main)  |           36.0+           | 10.3.1+ <br/>(10.3.1 needs hot-patch to prevent VCD cell crashes in multi-cell environments) |                                          [1.4.0](https://github.com/kubernetes-sigs/cluster-api/releases/tag/v1.4.0)                                           | <ul><li>1.25</li><ul><li>1.24</li><li>1.23</li><li>1.22</li><li>1.21</li><li>1.20</li></ul> | <ul><li>[1.4](https://github.com/vmware/cloud-director-named-disk-csi-driver/releases/tag/1.4)</li> <ul><li>[1.3.1](https://github.com/vmware/cloud-director-named-disk-csi-driver/releases/tag/1.3.1)</li><li>[1.3.0](https://github.com/vmware/cloud-director-named-disk-csi-driver/releases/tag/1.3.0)</li></ul> | <ul><li>[1.4.0](https://github.com/vmware/cloud-provider-for-cloud-director/releases/tag/1.4.0)</li><ul><li>[1.3.0](https://github.com/vmware/cloud-provider-for-cloud-director/releases/tag/1.3.0)</li><li>[1.2.0](https://github.com/vmware/cloud-provider-for-cloud-director/releases/tag/1.2.0)</li></ul> |
> ```

Should we also add CSI 1.3.2 as we have already released this?


<!-- Sent from Reviewable.io -->

File: README.md [hunk 1]
```diff
@@ -31,13 +31,14 @@ Cluster API versions:
...
-|                          | TKG versions        |
-|--------------------------|---------------------| 
-| CAPVCD v1beta1  (main)   | 1.6.1, 1.5.4, 1.4.3 |
-| CAPVCD v1beta1  (v1.0.2) | 1.6.1, 1.5.4, 1.4.3 |
-| CAPVCD v1beta1  (v1.0.1) | 1.6.1, 1.5.4, 1.4.3 |
-| CAPVCD v1beta1  (v1.0.0) | 1.5.4, 1.4.3        | 
-| CAPVCD v1alpha4 (v0.5.x) | 1.4.0, 1.3.1        |
+|                          | TKG versions                  |
+|--------------------------|-------------------------------| 
+| CAPVCD v1beta1  (main)   | 2.2, 2.1, 1.6.1, 1.5.4, 1.4.3 |
```
Comment (Anirudh9794) [inline README.md:36]: Can we mention the patch version also for TKG 2.2 and TKG 2.1?

File: README.md [hunk 2]
```diff
@@ -12,14 +12,14 @@ The version of Cluster API Provider Cloud Director and Installation that are com
...
+|  [main](https://github.com/vmware/cluster-api-provider-cloud-director/tree/main)  |           36.0+           | 10.3.1+ <br/>(10.3.1 needs hot-patch to prevent VCD cell crashes in multi-cell environments) |                                          [1.4.0](https://github.com/kubernetes-sigs/cluster-api/releases/tag/v1.4.0)                                           | <ul><li>1.25</li><ul><li>1.24</li><li>1.23</li><li>1.22</li><li>1.21</li><li>1.20</li></ul> | <ul><li>[1.4](https://github.com/vmware/cloud-director-named-disk-csi-driver/releases/tag/1.4)</li> <ul><li>[1.3.1](https://github.com/vmware/cloud-director-named-disk-csi-driver/releases/tag/1.3.1)</li><li>[1.3.0](https://github.com/vmware/cloud-director-named-disk-csi-driver/releases/tag/1.3.0)</li></ul> | <ul><li>[1.4.0](https://github.com/vmware/cloud-provider-for-cloud-director/releases/tag/1.4.0)</li><ul><li>[1.3.0](https://github.com/vmware/cloud-provider-for-cloud-director/releases/tag/1.3.0)</li><li>[1.2.0](https://github.com/vmware/cloud-provider-for-cloud-director/releases/tag/1.2.0)</li></ul> |
```
Comment (Anirudh9794) [inline README.md:17]: There seems to be a formatting issue with the unordered list in the list of kubernetes versions, CPI versions and CSI versions for main branch

**Review by Anirudh9794 — APPROVED** [review]
LGTM with some minor comments

**Pull request merged**